src/jvm/main/org/jetbrains/kotlinx/lincheck/Utils.kt

```diff
@@ -336,13 +346,20 @@ internal object InternalLincheckTestUnexpectedException : Exception()
  */
 internal class LincheckInternalBugException(cause: Throwable): Exception(cause)
 
-internal fun stackTraceRepresentation(stackTrace: Array<StackTraceElement>): List<String> {
-    return transformStackTraceBackFromRemapped(stackTrace).map { it.toString() }.filter { line ->
-        "org.jetbrains.kotlinx.lincheck.strategy" !in line
-                && "org.jetbrains.kotlinx.lincheck.runner" !in line
-                && "org.jetbrains.kotlinx.lincheck.UtilsKt" !in line
+internal inline fun <R> runInIgnoredSection(block: () -> R): R =
```

Comment:
There are now `runInIgnoredSection` and `invokeInIgnoredSection` functions. It's easy to call `runInIgnoredSection` in the transformation logic. Could you please make it impossible somehow?